demo/dlb-init.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+ENABLED_DLB_PF_PCIIDS=${ENABLED_DLB_PF_PCIIDS:-270b 2710 2714}
+NUMVFS=${NUMVFS:-4}
+
+bind() {
+    BSF=$1
+    DRIVER=$2
+
+    LOOP=0
```

Comment:
Why do we need to try binding five times?

Reply:
As I told, I do not know. Do we do just one time? I don't know why qat lib did five times.
https://github.com/intel/qatlib/blob/main/quickassist/utilities/service/qat_init.sh.in

Reply:
Well, this is your PR and your code now, so you should know. QAT code may do it for its own reason. This PR is about DLB.

Reply:
My point in general is...

If we do have a reason why we should not 'try binding five times', then wouldn't we have to have a same question about qatlib init script..? IMO, the only difference of the functions between qatlib init script and dlb initcontainer is what device we are binding/unbinding. The qatlib initscript does for qat devices, and we do for dlb deivces. Is there some other things that I missed? Please let me know. I have no idea.

So, what I mean is.. we can change it into one time or rather 100 times, but I do not know why we should not do five times as it was made for qatlib. If there is any reason why we need to do with other numbers, please let me know why. I just have no idea why.

Reply:
> If there is any reason why we need to do with other numbers, please let me know why. I just have no idea why.

Because this is a different device and one time could be enough for it? Did you tried to do it once?

Reply:
Aha, ok, one time would be enough then.!
Well, it always gets succeeded at the first attempt to bind/unbind.